Vizzuality/soils-revealed
# Fix the initial setup of the project

This PR fixes the initial setup of the project by:
- Adding a `.nvmrc` file to fix the version of node
- Removing the `url-loader` plugin and custom configuration to load images

## Testing instructions

For the first point, you can run `nvm use` (you need [nvm](https://github.com/nvm-sh/nvm) to be installed on your machine) and check whether correct node version is in use: `node -v` should return `v12.16.1`.

For the second point, you can try to load an image in any component by adding this line:
```html
<img src="test.png" alt="Test" />
```
Make sure to create an image called “test.png” in the `/public` folder.

## Pivotal Tracker

Not tracked.
